modulefiles/GDAS/hercules.intel.lua

```diff
@@ -56,20 +55,25 @@ load("libxaw/1.0.13")
 load("udunits/2.2.28")
 load("ncview/2.1.9")
 load("netcdf-cxx4/4.3.1")
-load("py-pybind11/2.11.0")
+load("json/3.10.5")
 --load("crtm/v2.4_jedi")
 load("contrib/0.1")
 load("noaatools/3.1")
 load("rocoto/1.3.5")
+load("prod_util/2.1.1")
 
-load("hpc/1.2.0")
-unload("python/3.10.13")
-unload("py-numpy/1.22.3")
-load("miniconda3/4.6.14")
-load("gdasapp/1.0.0")
+load("py-jinja2/3.0.3")
+load("py-netcdf4/1.5.8")
+load("py-pybind11/2.11.0")
+load("py-pycodestyle/2.11.0")
+load("py-pyyaml/6.0")
+load("py-scipy/1.11.3")
+load("py-xarray/2023.7.0")
+load("py-f90nml/1.4.3")
+load("py-pip/23.1.2")
 
 -- below is a hack because of cmake finding the wrong python...
-setenv("CONDA_PREFIX", "/work2/noaa/da/python/opt/core/miniconda3/4.6.14/envs/gdasapp/")
+prepend_path("PYTHONPATH", "/work2/noaa/da/python/gdasapp/wxflow/20240528/src")
```

Comment:
I wonder if this is needed? 
One suggestion would be to have wxflow as a submodule of gdasapp, and use that.
Adding wxflow to the installed stack is in progress and it will eliminate the need for both those things.

Reply:
I would be on board with wxflow being part of GDASApp (and JCB for that matter), but since that would result in modifications to modulefiles for all machines, I suggest we do that in a future PR.